packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -86,20 +102,35 @@ class ChatMessage extends UIComponent<ReactProps<ChatMessageProps>, ChatMessageS
   }
 
   public state = {
+    focused: false,
     isFromKeyboard: false,
   }
 
+  protected actionHandlers: AccessibilityActionHandlers = {
+    // prevents default FocusZone behavior, e.g., in ChatMessageBehavior, it prevents FocusZone from using arrow keys as navigation (only Tab key should work)
+    preventDefault: event => {
+      event.preventDefault()
+    },
+  }
+
   private handleFocus = (e: React.SyntheticEvent) => {
```

Comment:
lets remove `private` modifier from this method as well, to preserve local consistency. But totally agree, we need to have general grooming around this scope modifiers story